applications/API/endpoints.py

```
@@ -0,0 +1,500 @@
+    @extend_schema(
+        tags=["Insults"],
+        operation_id="get_random_insult",
+        auth=[],
+        responses=OptimizedInsultSerializer,
+        request=None,
+        parameters=[
+            OpenApiParameter(
+                name="nsfw",
+                type=OpenApiTypes.BOOL,
+                location=OpenApiParameter.QUERY,
+                description="Filter explicit content. `true` returns only NSFW insults; `false` returns only SFW. If omitted, both are returned.",
```

Pylint (3.3.7)

[convention] Line too long (146/113)

(C0301)